src/main/java/seedu/clinic/logic/commands/HelpCommand.java

```diff
@@ -3,19 +3,218 @@
 import seedu.clinic.model.Model;
 
 /**
- * Format full help instructions for every command for display.
+ * Format full help instructions for respective help command to display.
  */
 public class HelpCommand extends Command {
 
     public static final String COMMAND_WORD = "help";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows all command with"
+            + " respective utility description\n"
+            + "Example: " + COMMAND_WORD + "\n\nSpecific commands with respective input samples "
+            + "can be shown as well \n" + "Example: help find";
+    public static final String MESSAGE_TOO_MANY_ARGUMENTS = "You can only enter one command at a time\n\n"
+            + MESSAGE_USAGE;
+    public static final String MESSAGE_WRONG_ARGUMENT = "You can only enter command from one of the "
+            + "following keywords: add, clear, delete, exit, find, view, update";
 
     public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+    public static final String[] ALLOWED_ARGUMENT = new String[]{
```

Comment:
Perhaps this can be renamed to ALLOWED_ARGUMENTS or ALLOWED_ARGUMENT_LIST

Reply:
Thanks for pointing that out, will push another commit to fix this!